src/gatsby/src/pages/about/management-plans/approved.js

```diff
@@ -104,14 +106,22 @@ const ApprovedListPage = () => {
 
   const menuContent = queryData?.allStrapiMenu?.nodes || []
   const parks = queryData?.allStrapiProtectedArea?.nodes || []
+  parks.sort((a, b) => {
+    // compare protected area name including special characters
+    const parkA = replaceSpecialCharacters(a.protectedAreaName).toLowerCase()
```

Comment:
Is there a more efficient way to do this outisde the sort? 
e.g. maybe you could add an extra property to every object in the list prior to sorting